src/tabrmd.c

```diff
@@ -265,6 +265,58 @@ show_version (const gchar  *option_name,
     g_print ("tpm2-abrmd version %s\n", VERSION);
     exit (0);
 }
+/*
+ * Break the 'combined_conf' string up into the name of the TCTI and the
+ * config string passed to the TCTI. The combined_conf string is formatted:
+ * "name:conf" where:
+ * - 'name' is the name of the TCTI library in the form 'libtcti-name.so'.
+ *   Additionally the prefix 'libtcti-' and suffix '.so' may be omitted.
+ * - 'conf' is the configuration string passed to the TCTI. This is TCTI
+ *   specific.
+ * Bot the 'name' and 'conf' fields in this string are optional HOWEVER the
+ * if no semicolon is present in the combined_conf string it will be assumed
+ * that the whole string is the 'name'. To provide a 'conf' string to the
+ * default TCTI the first character of the combined_conf string must be a
+ * semicolon.
+ * If a field in the combined_conf string indicates a default value then
+ * the provided tcti_filename or tcti_conf will not be set. This is to allow
+ * defaults to be set by the caller and only updated here if we're changing
+ * them.
+ */
+int
```

Comment:
bool return? You only use 0 or -1 for return.

Reply:
sure